### ISSUE TITLE
finding & copying the "whodat_assignment" file

### ISSUE THREAD

**Issue by sydneytoth** [bytezen/IML400#1]
Hi guys! Not sure if anyone'll see this but... I'm having trouble just finding the file we're supposed to clone from the main class repository (and cloning it to my own "workshops" folder). 

When I go into the workshops folder and list the files, all I'm seeing is "whodat_01" & "whodat_02"...

<img width="551" alt="screen shot 2016-02-05 at 8 45 33 am" src="https://cloud.githubusercontent.com/assets/16657893/12852669/01a11c44-cbe5-11e5-96b0-e23be6b95abf.png">

I tried to clone the "whodat_02" file to my machine, just to see if that might be it, but again I'm apparently totally forgetting how to do this because:

<img width="563" alt="screen shot 2016-02-05 at 8 49 14 am" src="https://cloud.githubusercontent.com/assets/16657893/12852741/5f3da084-cbe5-11e5-82f3-0cf020807064.png">

I know these are super basic/probably dumb questions but if anyone knows what my glaring error is, please help a sista out--peace & blessings y'all

**Comment by tyl33t** [comment]
I don't believe you're supposed to use whodat as your directory, try
mainClassRepo and then do git pull!
Ty

**Comment by sydneytoth** [comment]
Thank you so much!! 

Still having issues with git pull from mainClassRepo, as it's saying it won't go through because changes to my whodat_01 from class would be overwritten, but I'm gonna try and play around with it/try to update those changes to the server and then hopefully this will work!

**Comment by tyl33t** [comment]
Yeah I just ran into the same problem. :/ Unfortunately I couldn't figure
it out, it kept aborting the git pull because of the files I changed and
that they weren't stashed or committed. You can try playing around, but as
a last resort I just had to delete all of the files and do this again:
git clone https://github.com/bytezen/IML400.git
Hope that helps!
Ty

**Comment by sydneytoth** [comment]
Yeah that's exactly what it kept saying for me, even after I tried to commit the changes it was saying needed to be committed... I'll probably end up doing the same thing lol--thank you so much for your help!!

**Comment by tyl33t** [comment]
No problem! Anytime!
Ty

**Comment by mpkoz** [comment]
Does the solution have anything to do with the git checkout master command?
I remember doing that a couple times in class.

**Comment by jordanmfowler** [comment]
I think you can pull it and when you do, you can just save the file as V2 and change the name as you clone it. This way you can keep both copies.
